lib/rihanna/job.ex

```diff
@@ -59,6 +60,15 @@ defmodule Rihanna.Job do
   end
   ```
 
+  You can define a `priority/0` function which will be called if no priority
+  is set when a job is enqueued. It should return a single integer. A priority
+  of 1 will be the highest priority, with larger numbers being run after.
+  If you don't define this callback it will default to the lowest priority, 50.
```

Comment:
50 is not the lowest priority. You could set a priority of 100 if you like.

Reply:
Why not just 0 as default, negative for high priority and positive for low priority?

Reply:
Yes, that's true, you could set a number bigger than 50. Nothing caps that.

I was following chanks/que priority behavior:

```
# We use the Linux priority scale - a lower number is more important.
self.priority = 10
```

Reply:
They default the priority in the DB to `1`, though. Maybe that's a better idea than trying to make it lower priority.

Reply:
Makes sense to follow what Que is doing